core/circuitbreaker/rule_manager.go

```diff
@@ -374,5 +374,8 @@ func IsValid(r *Rule) error {
 	if r.Strategy == ErrorRatio && r.Threshold > 1.0 {
 		return errors.New("invalid error ratio threshold (valid range: [0.0, 1.0])")
 	}
+	if r.StatSlidingWindowBucketCount != 0 && r.StatIntervalMs%r.StatSlidingWindowBucketCount != 0 {
+		logging.Warn("The following must be true: StatIntervalMs % StatSlidingWindowBucketCount == 0. StatSlidingWindowBucketCount will be replaced by 1", "rule", r)
```

Comment:
Print the log according to the specification  https://github.com/alibaba/sentinel-golang/pull/293

Reply:
Resolved. Thanks.